### DISCUSSION TITLE
Underscore messages on 7-segment LED

### DISCUSSION THREAD

**Discussion by barbeque** [keirf/flashfloppy#437] [Q&A] [answered]
Hope everything is going well with you and yours. I am trying to use FlashFloppy 3.23 on a Gotek with an NEC PC-6601SR. I’ve set the drive to shugart mode in FF.CFG, and it does seem to load some of my test images converted to HFE format. This technique worked well for me on a Sharp X1 and PC-8801.  

However, I’m suspicious that the floppy controller is acting up in this machine, as the original drive’s motor (it's a Chinon F-323, which I can find no info on) seems to run forever and not allow the computer to boot even without a disk inserted. And with the Gotek installed, the access LED is frequently blinking even in what seems like inappropriate places, and the USB stick’s access light is constantly blinking. 

I noticed while trying to diagnose this behaviour that the LED display on the Gotek sometimes displays codes like “0_”, "42_" and “36_”. Do these mean anything or is it spurious?

I also have an OLED display I could hook up, and I could also enable logging, if that will give more info. 

![8B74FBB1-A34E-44C2-8F37-A1441B13B63B](https://user-images.githubusercontent.com/423811/109361766-d6b28680-7846-11eb-84ec-6796e950aec9.jpeg)
![996FB330-6FA7-4B52-9DF7-ECF3ACBB8E24](https://user-images.githubusercontent.com/423811/109361773-da460d80-7846-11eb-81b3-4b1661310084.jpeg)

I believe the power supply is functioning properly (measured exactly 5.00V at the Gotek power plug.) The floppy drive controller is an NEC µPD765, although there is also some SRAM and a custom QFP on the floppy controller board that I haven't yet been able to figure out the function of.

I also haven't used a very recent version of FlashFloppy (I think the last one I used was the version I did the pc98 PR for) so if this is a new feature (it looks like `led_7seg_update_track` might be responsible?) then please ignore this :)
 
Thanks again for the great firmware!

**Comment by keirf** [comment, marked as answer]
The  display with underscore is format `NN_` where NN is cylinder number and the position of the underscore indicates lower or upper head. It's a glorified track counter.

**Reply by barbeque** [reply]
Very clever, thank you!

**Reply by NigelPearson** [reply]
Could this possibly be documented?
(like, at the bottom of
https://github.com/keirf/flashfloppy/wiki/Error-Messages
where the other non-error code F-F is documented)
